config.vala.in

```diff
@@ -0,0 +1,3 @@
+namespace Build {
+    public const string PROJECT_NAME = @PROJECT_NAME@;
```

Comment:
This config is only ever used once, in `src/Tweaks.vala`. I suggest removing all config related code and just hardcode its 1 use in `Tweaks.vala`. This config is overkill otherwise.